### ISSUE TITLE
Failing TravisCI tests

### ISSUE THREAD

**Issue by dfreerksen** [spree-contrib/spree-product-assembly#111]
With the most recent commit, Travis CI is failing for all versions of Ruby. It is getting an error similar to the one at the bottom of this issue.

This issue is coming from this line https://github.com/spree-contrib/spree-product-assembly/blob/master/app/serializers/spree/wombat/assembly_shipment_serializer.rb#L35 If you remove the `if` everything runs as expected. Is the `if` even needed? `lib/spree_product_assembly/engine.rb` is already checking if `Spree::Wombat::Engine` is loaded before it loads the serializers.

Maybe it would be better to only load `spree_wombat` in development or test environments only?

I'm not using Wombat so I'm not sure what I am saying is valid or correct. That is why this is an issue instead of a PR. Any insight on this would help.

```
Coverage report generated for RSpec to /Users/dfreerksen/ruby/projects/spree-product-assembly/coverage. 82 / 172 LOC (47.67%) covered.
/Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:481:in `load_missing_constant': Unable to autoload constant Spree::Wombat::AssemblyShipmentSerializer, expected /Users/dfreerksen/ruby/projects/spree-product-assembly/app/serializers/spree/wombat/assembly_shipment_serializer.rb to define it (LoadError)
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:180:in `const_missing'
	from /Users/dfreerksen/ruby/projects/spree-product-assembly/spec/serializers/spree/wombat/assembly_shipment_serializer_spec.rb:5:in `<module:Wombat>'
	from /Users/dfreerksen/ruby/projects/spree-product-assembly/spec/serializers/spree/wombat/assembly_shipment_serializer_spec.rb:4:in `<module:Spree>'
	from /Users/dfreerksen/ruby/projects/spree-product-assembly/spec/serializers/spree/wombat/assembly_shipment_serializer_spec.rb:3:in `<top (required)>'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:241:in `load'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:241:in `block in load'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activesupport-4.1.9/lib/active_support/dependencies.rb:241:in `load'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `block in load_spec_files'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `each'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `load_spec_files'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:96:in `setup'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:84:in `run'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:69:in `run'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:37:in `invoke'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/rspec-core-3.1.7/exe/rspec:4:in `<top (required)>'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/bin/rspec:23:in `load'
	from /Users/dfreerksen/.rbenv/versions/2.1.2/bin/rspec:23:in `<main>'
```

**Comment by JDutil** [comment]
Yea I don't think the if is necessary but probably need to add back the AMS dependency as a development dependency.

**Comment by JDutil** [comment]
Fixed by 5c8b9bf9072be946691e8cef60ce08272c2dd84a
